src/nylo/tokens/keyword.py

```diff
@@ -1,61 +1,86 @@
+# This file is a part of nylo
```

Comment:
keyword.py should not have the copyright. Only file beggining with __ shall have them.